osu.Game/Overlays/Dashboard/Friends/FriendsOnlineStatusControl.cs

```diff
@@ -5,20 +5,19 @@
 using System.Linq;
 using osu.Game.Users;
 
-namespace osu.Game.Overlays.Home.Friends
+namespace osu.Game.Overlays.Dashboard.Friends
 {
     public class FriendsOnlineStatusControl : OverlayStreamControl<FriendsBundle>
     {
         protected override OverlayStreamItem<FriendsBundle> CreateStreamItem(FriendsBundle value) => new FriendsOnlineStatusItem(value);
 
         public void Populate(List<User> users)
         {
-            var userCount = users.Count;
-            var onlineUsersCount = users.Count(user => user.IsOnline);
+            Clear();
 
-            AddItem(new FriendsBundle(FriendsOnlineStatus.All, userCount));
-            AddItem(new FriendsBundle(FriendsOnlineStatus.Online, onlineUsersCount));
-            AddItem(new FriendsBundle(FriendsOnlineStatus.Offline, userCount - onlineUsersCount));
+            AddItem(new FriendsBundle(FriendsOnlineStatus.All, users));
```

Comment:
i don't get this. why are we making three copies of users? why is this not a filter control like every other overlay implementation? at very least, why aren't these live linq queries and instead creating copies of the list?